src/scripts/components/stories/story-gallery-item/story-gallery-item.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {FunctionComponent} from 'react';
+import cx from 'classnames';
+
+import styles from './story-gallery-item.module.styl';
+
+interface Props {
+  children: React.ReactNode[];
+  currentIndex: number;
+  showLightbox: boolean;
+}
+
+const StoryGalleryItem: FunctionComponent<Props> = ({
+  children,
+  currentIndex,
+  showLightbox
+}) => {
+  const containerWidth = children.length * 100;
+  const itemWidth = 100 / children.length;
+  const imgClasses = cx(
+    styles.slider,
+    showLightbox && styles.lightboxStoryGallery,
+    children.length > 1 && styles.transition
+  );
+
+  return (
+    <div
+      className={imgClasses}
+      style={{
+        width: `${containerWidth}%`,
```

Comment:
I would suggest we create custom CSS variables and control the value using `document.documentElement.style.setProperty(
      '--container-width'),` - that why we would have better separation of concerns

Reply:
As discussed I think we should keep it as it is to stay consistent. Also I think a global css variable is not needed for this case.